smart-home-provider/cloud/auth-provider.js

```diff
@@ -304,6 +302,53 @@ Auth.registerAuth = function(app) {
 // &grant_type=authorization_code
 
 
+/**
+ * Decode authorization into client_id and client_secret.
+ * @return {{}}
+ * {
+ *   client_id: "CLIENT_ID",
+ *   client_secret: "CLIENT_SECRET"
+ * }
+ */
+function getHeaderAuthorization(req){
+  let headerAuthor = {};
+    try{
```

Comment:
Small nitpicking around spacing. The `try` should be the same level indentation as the `let` above it, and there should be spaces after each word, ie. `if (...) {`

Reply:
Oh, I'll correct this issue.